src/user/room.ts

```diff
@@ -428,23 +428,33 @@ export class Room extends Accessory implements Sayable {
     textOrContactOrFileOrUrl : string | Contact | FileBox | UrlLink,
     mention?                 : Contact | Contact[],
   ): Promise<void> {
+
+    const replyToList: Contact[] = [].concat(mention as any || [])
```

Comment:
use `[...mention]` should be better.

Reply:
but the replyToList is original code, and what if mention is undefine?

Reply:
`[...(undefined||[])]`